File: LPS_Service/Models/Account/DepositAccountModel.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Web;

namespace LPS_Service.Models.Account
{
    public class DepositAccountFilterModel
    {
        [Required]
        public int startPage { get; set; }
        [Required]
        public int limitPage { get; set; }
        [Required]
        public string cif { get; set; }
        public string sortBy { get; set; }
        public bool ascending { get; set; }
        public string productGroup { get; set; }
    }
    public class GetDepositAccountModel
    {
        public SpDepositAccountModel[] accounts { get; set; }
        public int totalItems { get; set; }
    }
}

File: LPS_Angular/src/app/models/Account/DepositAccountModel.ts
export class DepositAccountModel {
    ACCTNO: string;
    ACTYPE: string;
    PRODUCT_CODE: string;
    PRODUCT_GROUP: string;
    BALANCE: number;
    INT_RATE: number;
    ISSUE_DATE: string;
    MATURITY_DATE: string;
}
export class GetDepositAccountModel {
    accounts: DepositAccountModel[];
    totalItems: number;
}
export class DepositAccountFilterModel {
    startPage: number;
    limitPage: number;
    cif: string;
    sortBy: string;
    ascending: boolean;
    productGroup: string;
}
export class DepositAccountSortingModel {
    accountNo: boolean;
    acType: boolean;
    productCode: boolean;
    productGroup: boolean;
    balance: boolean;
    intRate: boolean;
    issueDate: boolean;
    maturityDate: boolean;
}
